core/server/config/utils.js

```diff
@@ -69,6 +70,21 @@ exports.getContentPath = function getContentPath(type) {
 };
 
 /**
+* Check if the URL in config has a protocol and sanitise it if not including a warning that it should be changed
+*/
+exports.sanitiseUrlProtocol = function sanitiseUrlProtocol() {
+    var url = this.get('url');
+
+    if (url.match(/^https?:\/\//i)) {
+        return;
+    } else {
+        this.set('url', 'http://' + url);
+        console.log(chalk.red('Watch out! Your URL in the config should have a protocol! E.g. "http://my-ghost-blog.com"'));
+        console.log(chalk.white('Current URL: ') + chalk.cyan(url));
```

Comment:
Would you just prefer to have the stdout for this bundled in `core/server/ghost-server`? Or not output at all?

Here, specifically, I output the url the user has set up in the config and set the sanitised value of this url as the new url to work with.

The user will then first see, how he set up the url and then (coming from `ghost-server.js`) how it should look like.

Here an example:

![image](https://cloud.githubusercontent.com/assets/8037602/26192473/c56e97be-3bec-11e7-8aad-c1b420ea4382.png)